guides/deployment/releases.md

```diff
@@ -19,7 +19,7 @@ $ elixir -v
 
 If you are not familiar with Elixir releases yet, we recommend you to read [Elixir's excellent docs](https://hexdocs.pm/mix/Mix.Tasks.Release.html) before continuing.
 
-Once that is done, you can assemble a release by going through all of the steps in our general [deployment guide](deployment.html) with `mix release` at the end. Let's recap.
+Once that is done, you can assemble a release by going through all of the steps in our general [deployment guide](https://hexdocs.pm/phoenix/deployment.html) with `mix release` at the end. Let's recap.
```

Comment:
Not sure, if this is right approach. the link is referencing internal guide, and all internal guides are linked to each other, for example link to Up and Running guide above. and it is working fine, so it seems there is another underlying bug here

Reply:
also, i just created the bug, not sure what maintainer team's take on this